### PR TITLE
Add console_batt for wn1980 console 

### PR DESCRIPTION
**Pull request by pantonvich**

### CODE REVIEW

**Comment by pantonvich** [comment]
https://github.com/home-assistant/core/issues/108922

**Review by pvizeli — APPROVED** [review]

**Comment by pvizeli** [comment]
Need fix CI

**Comment by pantonvich** [comment]
@pvizeli sorry not sure what I'm doing with github - I think this is what you want with "Need fix CI" - and not sure my commits on my branch get pushed back here - I think I need to delete pull 144 that I did in my main branch

**Comment by pantonvich** [comment]
@pvizeli this should pass all the workflow now

File: misc/example.data [hunk 1]
```diff
@@ -1,4 +1,4 @@
...
+<MultiDictProxy('PASSKEY': '345544D8EAF42E1B8824A86D8250D5A3', 'stationtype': 'GW2000A_V2.1.5', 'runtime': '188738', 'dateutc': '2022-07-15 15:26:14', 'tempinf': '78.80', 'humidityin': '53', 'baromrelin': '27.885', 'baromabsin': '27.885', 'tempf': '87.08', 'humidity': '34', 'winddir': '289', 'windspeedmph': '2.91', 'windgustmph': '3.13', 'maxdailygust': '12.53', 'solarradiation': '530.62', 'uv': '4', 'rrain_piezo': '0.000', 'erain_piezo': '0.000', 'hrain_piezo': '0.000', 'drain_piezo': '0.000', 'wrain_piezo': '0.000', 'mrain_piezo': '0.000', 'yrain_piezo': '0.000', 'ws90cap_volt': '5.4', 'ws90_ver': '119', 'wh90batt': '2.74', 'console_batt': '2.5', 'freq': '868M', 'model': 'GW2000A')>
```
Comment (pvizeli) [inline misc/example.data:4]: Can you add your device type into? now we have only a simple copy. Make not much sense

Comment (pvizeli) [reply misc/example.data:4]: You can also ditch it, it's more for knowing what station send over, i.e. the EasyWeatherV1.4.9 or GW2000A_V2.1.5 - so you can also just ditch it if you not have somethings from a new station type

Comment (pantonvich) [reply misc/example.data]: ok I've added my example that I found in the device diagnostics json text download - mine doesn't have a passkey and was listed as MultiDict and not MultiDictProxy like the other examples

**Review by pvizeli — APPROVED** [review]

**Comment by pantonvich** [comment]
@pvizeli What would you like me to do? It looks like the test array counts are out of sync from the soil pull 145? Should I just update the counts again or create a test set for this one value console_batt for the wn1980 or add/keep it with GW2000 dataset?

**Comment by pvizeli** [comment]
> test set for this console_batt for the wn1980

That would be nice if you could do that, more test make it more stable on updates 👍 but I'm also fine if you just fix the tests

**Pull request merged**